<launch>

  <group ns="one">

      <node pkg="navigation_point" name="navigation_point_node" type="navigation_point.py">
          <remap from="in_topic" to="yellow_mask"/>
          <remap from="out_topic" to="yellow_centroid"/>
      </node>

  </group>

</launch>
